index.d.ts

```diff
@@ -52,6 +52,10 @@ declare module 'plaid' {
 
   type AssetReportRefreshOptions = AssetReportCreateOptions;
 
+  interface WebhookOptions {
```

Comment:
Please ignore all lines except for the ones that I have marked with comments saying "here". I just prettified things